node.js

```diff
@@ -53,10 +53,12 @@ module.exports = {
     'no-undefined': off,
     'no-use-before-define': [ error, { functions: false, classes: true } ],
     'object-curly-spacing': [ error, always ],
-    'array-bracket-spacing': [ error, always, { objectsInArrays: false } ],
```

Comment:
as per discussion, previous added rule makes for some weird looking code, so i'm tightening it